src/components/Results/Results.tsx

```diff
@@ -67,45 +129,55 @@ const Results: React.FC<Props> = ({ measureFile, patientFile, htmls }) => {
               </Grid>
             );
           })}
-      </Grid>
-      <Grid container item xs={12} direction="row">
-        <Grid item xs>
-          <Grid container item xs direction="row">
-            {results && <h2>JSON:</h2>}
-            {results && (
-              <IconButton
-                onClick={() => {
-                  fileDownload(
-                    JSON.stringify(results, null, 2),
-                    measureFile.name?.includes('.json')
-                      ? `results-${measureFile.name}`
-                      : `results-${measureFile.name}.json`
-                  );
-                }}
-              >
-                <GetApp fontSize="small" />
-              </IconButton>
-            )}
-          </Grid>
-          {results && <ReactJson src={results} enableClipboard={true} theme="shapeshifter:inverted" collapsed={2} />}
-          {patientFile && <h2>Patient Bundle:</h2>}
-          {patientFile && (
-            <ReactJson src={patientFile} enableClipboard={true} theme="shapeshifter:inverted" collapsed={2} />
-          )}
-        </Grid>
-        <Grid item xs>
-          {results &&
-            htmls &&
-            htmls.map(html => {
-              return (
-                <div key={html.groupId} className={classes.highlightedMarkup}>
-                  <h2>HTML:</h2>
-                  {parse(html.html)}
-                </div>
-              );
-            })}
-        </Grid>
-      </Grid>
+      </>
+    );
+  };
+
+  const shouldDisplayHTML = () => {
+    return htmls.length > 0;
+  };
+
+  const shouldDisplayAccordion = () => {
+    return outputType === 'gapsInCare';
+  };
+
+  const shouldDisplayTabular = () => {
+    return outputType === 'measureReports' && calculationOptions.reportType === 'individual';
+  };
+
+  const shouldDisplayTabs = () => {
+    return shouldDisplayHTML() || shouldDisplayAccordion() || shouldDisplayTabular();
+  };
```

Comment:
I like the use of these functions. Makes it easy to see when things should/shouldn't be displayed, and easy to extend in the future!